### ISSUE TITLE
Rename "Websites" to "Links"

### ISSUE THREAD

**Issue by dev-101** [matomo-org/matomo#16356]
**Acquisition > Websites**

This is only a suggestion, in order to better match Google ("**Links**") and Microsoft ("**Backlinks**").

**Comment by tsteur** [comment]
@dev-101 are you referring to the Search Consoles? Cause in the analytics tool it's eg named "Referrals".

**Comment by dev-101** [comment]
Yes, I haven't looked at Analytics, sorry. Well, "referrals" also sounds good.

**Comment by mattab** [comment]
@dev-101 do you maybe have any thoughts on why `Referrals` would be better than `Websites`?

**Comment by tsteur** [comment]
@mattab one advantage is that it works better for mobile apps etc as it is more generic. At the same time it's a disadvantage for regular web analytics because it is less clear what it is

**Comment by Elreywad** [comment]
I'll rename it for you
